app/controllers/api/v1/api_controller.rb

```diff
@@ -0,0 +1,14 @@
+module Api
+  module V1
+    class ApiController < ApplicationController
+      rescue_from ActiveRecord::RecordInvalid, with: :show_errors
```

Comment:
You will be using this in a lot of controllers. Move it to ApplicationController so you don't have to repeat code everywhere.